hazelcast-jclouds/src/test/java/com/hazelcast/jclouds/integration/LiveComputeServiceBuilder.java

```diff
@@ -0,0 +1,131 @@
+package com.hazelcast.jclouds.integration;
+
+import com.google.common.base.Predicates;
+import org.jclouds.ContextBuilder;
+import org.jclouds.compute.ComputeService;
+import org.jclouds.compute.ComputeServiceContext;
+import org.jclouds.compute.domain.NodeMetadata;
+import org.jclouds.compute.domain.Template;
+import org.jclouds.compute.domain.TemplateBuilder;
+import org.jclouds.compute.predicates.NodePredicates;
+
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+import static com.google.common.base.Predicates.not;
+import static com.google.common.collect.Iterables.concat;
+import static com.google.common.collect.Iterables.getOnlyElement;
+import static com.hazelcast.jclouds.integration.AbstractLiveTest.AWS_IDENTITY;
+import static com.hazelcast.jclouds.integration.AbstractLiveTest.AWS_PROVIDER;
+import static com.hazelcast.jclouds.integration.AbstractLiveTest.AWS_CREDENTIAL;
+import static org.jclouds.compute.options.TemplateOptions.Builder.userMetadata;
+import static org.jclouds.compute.predicates.NodePredicates.inGroup;
+
+
+public class LiveComputeServiceBuilder {
+
+    private  ComputeService computeService;
+    private  List<NodeMetadata> nodes = new ArrayList<NodeMetadata>();
+    private  String provider;
+
+
+    public static final String GROUP_NAME1 = "livetest-group1";
+    public static final String GROUP_NAME2 = "livetest-group2";
+
+    public static final String AWS_REGION1 = "us-east-1";
+    public static final String AWS_REGION2 = "us-west-2";
+
+    public static final Map.Entry<String, String> tag1 =
+            new AbstractMap.SimpleImmutableEntry<String, String>("Owner", "DbAdmin");
+    public static final Map.Entry<String, String> tag2 =
+            new AbstractMap.SimpleImmutableEntry<String, String>("Stack", "Test");
+    public static final Map.Entry<String, String> tag3 =
+            new AbstractMap.SimpleImmutableEntry<String, String>("Type", "Micro");
+
+    public LiveComputeServiceBuilder(String provider) {
+        this.provider = provider;
+        init();
+    }
+
+    public void init() {
+        if (computeService == null) {
+            computeService = initComputeService(AWS_PROVIDER, AWS_IDENTITY, AWS_CREDENTIAL);
```

Comment:
`LiveComputeServiceBuilder` feels more Test Util class so using Util in the name might be more readable. Additionally, accepting `initComputeService` parameters from outside will make this class independent of CloudProvider...